src/main.rs

```diff
@@ -518,13 +520,15 @@ impl Fastmod {
     }
 
     fn run_fast(
+        threads: &str,
```

Comment:
doesn't seem like threads is the most important thing for callers to worry about. 1) why is it the new first argument? 2) why isn't it optional?

Reply:
I thought this was a private function abstracted away via the CLI. The default would be the number of CPUs which is detected at run time.